docs/tools-offline-instructions/dependencies.json

```diff
@@ -0,0 +1,27 @@
+{
+    "vscode_extensions": {
+        "sapse.sap-ux-fiori-tools-extension-pack": "latest"
+    },
+    "node_modules": {
```

Comment:
`@sap/ux-cds-odata-language-server-extension` is also required for CAP CDS projects. CAP CDS VSCode extension loads this dynamically from NPM registry and the registry URL can be overwritten with VSCode setting  `cds.contributions.registry`.

Reply:
Added it to `docs/tools-offline-instructions/info/globalModules.json`